test/integration/I18n_test.rb

```diff
@@ -183,7 +183,7 @@ class I18nTest < ActionDispatch::IntegrationTest
       follow_redirect!
 
       get '/wiki'
-      assert_select 'th', I18n.t('wiki._wikis.likes')
+      assert_select 'hr#first'
```

Comment:
@mittal-parth Since we are testing for translation strings, I don't think we should be using HTML elements that don't use it. Maybe we could add `Published` in the translation file and test that

https://github.com/publiclab/plots2/blob/0aaf28cc36ac2e0b09613cc1dbe41c3457cb1025/app/views/wiki/_wikis.html.erb#L20

Reply:
Thanks for this -- tests pass currently but let's resolve this before we merge if that's ok @mittal-parth ?

Reply:
Oh I had also pointed that out before 😅  No problem, fixed that.
Since we are at it, should I also add translation to _Based on what you are following at_, _Discover more...._, _Manage your subscription settings_, _You haven't subscribed to any topic....._? @TildaDares @jywarren 
(Sorry for the trouble!)

Reply:
@mittal-parth Sure you can do that